core-web/libs/dotcms-models/src/lib/dot-workflow-action.model.ts

```diff
@@ -24,6 +24,19 @@ export interface DotCMSWorkflowAction {
     showOn: string[];
     actionInputs: DotCMSWorkflowInput[];
     metadata?: Record<string, string>;
+    hasArchiveActionlet?: boolean;
+    hasCommentActionlet?: boolean;
+    hasDeleteActionlet?: boolean;
+    hasDestroyActionlet?: boolean;
+    hasMoveActionletActionlet?: boolean;
+    hasMoveActionletHasPathActionlet?: boolean;
+    hasOnlyBatchActionlet?: boolean;
+    hasPublishActionlet?: boolean;
+    hasPushPublishActionlet?: boolean;
+    hasResetActionlet?: boolean;
+    hasSaveActionlet?: boolean;
+    hasUnarchiveActionlet?: boolean;
+    hasUnpublishActionlet?: boolean;
```

Comment:
Can we check this dynamically? This seems unmaintainable in the long term

Reply:
Let me ask to backend this.

Reply:
But, now the actions, can have sub-actions
![CleanShot 2024-11-21 at 13 47 44@2x](https://github.com/user-attachments/assets/cb035c59-95e8-4cbb-823d-7a7e14c373c1)

Reply:
Asking about this topic to BE.